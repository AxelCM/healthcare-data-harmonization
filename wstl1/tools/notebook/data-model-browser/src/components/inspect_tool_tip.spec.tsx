/**
 * @jsx React.createElement
 * @jsxFrag React.Fragment
 */
// Copyright 2020 Google LLC.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// tslint:disable-next-line:enforce-name-casing
import * as React from 'react';
import {shallow} from 'enzyme';
import {InspectTooltip} from './inspect_tool_tip';

describe('InspectTooltip', () => {

  it('should render correctly', () => {
    const wrapper = shallow(
      <InspectTooltip title='Test Tool Tip'>
        <button id='testChild' type='submit'>
            Hello World
        </button>
      </InspectTooltip>);
    expect(wrapper).toMatchSnapshot();
  });
});
