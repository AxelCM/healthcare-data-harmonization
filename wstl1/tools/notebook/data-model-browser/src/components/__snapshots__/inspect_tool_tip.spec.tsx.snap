// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InspectTooltip should render correctly 1`] = `
<WithStyles(ForwardRef(Tooltip))
  PopperProps={
    Object {
      "popperOptions": Object {
        "modifiers": Object {
          "arrow": Object {
            "element": null,
            "enabled": false,
          },
        },
      },
    }
  }
  classes={
    Object {
      "popper": "makeStyles-popper-2",
      "tooltip": "makeStyles-tooltip-3",
      "tooltipPlacementLeft": "makeStyles-tooltipPlacementLeft-4",
    }
  }
  placement="left"
  title={
    <React.Fragment>
      Test Tool Tip
      <span />
    </React.Fragment>
  }
>
  <button
    id="testChild"
    type="submit"
  >
    Hello World
  </button>
</WithStyles(ForwardRef(Tooltip))>
`;
